pandas/core/window/rolling.py

```diff
@@ -494,9 +494,7 @@ def _get_cython_func_type(self, func: str) -> Callable:
 
         Variable algorithms do not use window while fixed do.
         """
-        if self.is_freq_type or isinstance(self.window, BaseIndexer):
-            return self._get_roll_func(f"{func}_variable")
-        return partial(self._get_roll_func(f"{func}_fixed"), win=self._get_window())
+        return self._get_roll_func(f"{func}_variable")
```

Comment:
This shouldn't change. This will incur a performance hit for fixed window aggregations since they use different algorithms

Reply:
Thanks for the feedback, reverted the change